dynamic_programming/catalan_numbers.cpp

```diff
@@ -1,75 +1,78 @@
-/** Print all the Catalan numbers from 0 to n, n being the user input.
-
- * A Catalan number satifies the following two properties:
+/**
```

Comment:
could you add some information like author, the brief tag as well as the file tag?

Reply:
Please have a look at 5cc72eb7c3c847deb3ac1741c3996767b9bdac26.

The `@author` tag seams redundant to me (one can always use blame). If you insist on it please let me know or just commit it.